`pip install` command in CONTRIBUTING does not install `psycopg` dependency required to run tests

I created and activated a virtual environment then installed dependencies as directed in `CONTRIBUTING.md`, but tests fail to run due to missing `psycopg` dependency:

```
$ pip install pre-commit -e .["dev,test"]
$ python -m pytest --cov titiler.pgstac --cov-report term-missing

ImportError while loading conftest '/path/to/titiler-pgstac-test/tests/conftest.py'.
tests/conftest.py:7: in <module>
    import psycopg
E   ModuleNotFoundError: No module named 'psycopg'
```

The `psycopg` extra needs to be added to the `pip` install for the instructions to work as directed.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -6,8 +6,8 @@ Issues and pull requests are more than welcome: https://github.com/stac-utils/ti
 
 ```bash
 $ git clone https://github.com/stac-utils/titiler-pgstac.git
-$ cd titiler
-$ pip install pre-commit -e .["dev,test"]
+$ cd titiler-pgstac
+$ pip install pre-commit -e .["dev,test,psycopg"]
 ```
 
 You can then run the tests with the following command:

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -7,7 +7,7 @@ Issues and pull requests are more than welcome: https://github.com/stac-utils/ti
 ```bash
 $ git clone https://github.com/stac-utils/titiler-pgstac.git
 $ cd titiler-pgstac
-$ pip install pre-commit -e .["dev,test,psycopg"]
+$ python -m pip install -e ".[dev,test,psycopg]"
 ```
 
 You can then run the tests with the following command:

